src/main/java/mat/client/measure/ManageMeasureSearchView.java

```diff
@@ -36,435 +32,352 @@
-		
-	}
-
-	@Override
-	public HasClickHandlers getClearButton() {
-		return clearButton;
-	}
-
-	@Override
-	public MessageAlert getErrorMessagesForTransferOS() {
-		return errorMessagesForTransferOS;
-	}
-
-	@Override
-	public HasClickHandlers getTransferButton() {
-		return transferButton;
-	}
-
-	public ManageMeasureSearchModel getData() {
-		return data;
-	}
-
-	public void setData(ManageMeasureSearchModel data) {
-		this.data = data;
-	}
-
-	@Override
-	public void setAdminObserver(AdminObserver adminObserver) {
-		measureSearchView.setAdminObserver(adminObserver);
-	}
-
-	@Override
-	public HasClickHandlers getAdminSearchButton() {		
-		return searchWidgetBootStrap.getGo();
-	}
-
-	@Override
-	public HasValue<String> getAdminSearchString() {		
-		return searchWidgetBootStrap.getSearchBox();
-	}
-
-	@Override
-	public Button getCreateMeasureButton() {
-		return createMeasureButton;
-	}
-	
-	@Override
-	public String getSelectedOption() {
-		return null;
-	}
-
-	@Override
-	public EditConfirmationDialogBox getDraftConfirmationDialogBox() {
-		return draftConfirmationDialogBox;
-	}
-
-	@Override
-	public MessageAlert getSuccessMessageDisplay() {
-		return successMessages;
-	}
-
-	@Override
-	public void resetMessageDisplay() {
-		getErrorMessageDisplayForBulkExport().clearAlert();
-		getErrorMessageDisplay().clearAlert();
-		getErrorMeasureDeletion().clearAlert();
-		getSuccessMeasureDeletion().clearAlert();
-		getSuccessMessageDisplay().clearAlert();
-	}
-
-	@Override
-	public CustomCheckBox getCustomFilterCheckBox() {
-		return measureSearchFilterWidget.getMeasureCustomCheckBox();
-	}
-	
-	@Override
-	public AdvancedSearchPillPanel getSearchPillPanel() {
-		return pillPanel;
-	}
-	
-	@Override
-	public void resetDisplay() {
-		this.getErrorMessageDisplay().clearAlert();
-		this.measureSearchFilterWidget.getSearchInput().setValue("");
-		this.measureSearchFilterWidget.getMeasureCustomCheckBox().setValue(true);
-		this.measureSearchFilterWidget.getAdvancedSearchPanel().resetDisplay(true);
-		this.measureSearchFilterWidget.getSearchInput().getElement().focus();
-		this.measureSearchFilterWidget.setSelectedFilter(SearchWidgetWithFilter.MY);
-	}
+    Button createMeasureButton = new Button("New Measure");
+
+    Button createCompositeMeasureButton = new Button("New Composite Measure");
+
+    CellTable<ManageMeasureSearchModel.Result> table;
+
+    private final MessageAlert errorMessagesForTransferOS = new ErrorMessageAlert();
+
+    private MessageAlert errorMeasureDeletion = new ErrorMessageAlert();
+
+    private final Button clearButton = new Button("Clear All");
+
+    private final MessageAlert errorMessages = new ErrorMessageAlert();
+
+    private final MessageAlert errorMessagesForBulkExport = new ErrorMessageAlert();
+
+    private ManageMeasureSearchModel data = new ManageMeasureSearchModel();
+
+    private final FormPanel form = new FormPanel();
+
+//    public VerticalPanel measureVpanel = new VerticalPaGrid508nel();
```

Comment:
This looks like we have commented it out. Do we still need it?

Reply:
Ups, I missed that. Will remove